format-specs/geoparquet.md

```diff
@@ -79,6 +80,15 @@ This is the binary format that the geometry is encoded in. The string 'WKB' to r
 [Well Known Binary](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry#Well-known_binary) is the only current option, but future versions
 of the spec may support alternative encodings. This should be the ["standard"](https://libgeos.org/specifications/wkb/#standard-wkb) WKB representation.
 
+#### edges
+
+This attribute indicates the coordinate system to be applied when interpreting the edges. Available values are:
```

Comment:
I find the current use of "coordinate system" in "indicates the coordinate system to be applied when interpreting .." a bit confusing, since that word is closely related to coordinate reference system, and this field is independent from the CRS. 
I know that strictly speaking this is correct (since it's about using a cartesian vs spherical coordinate system), but since both can be used while using a geographical coordinate system, I think we might need to explain this a bit more to avoid confusion.

A first attempt:

> This attribute indicates how to interpret the edges of the geometries: whether the line between two points is a straight Cartesian line or the shortest line on the sphere (geodesic line).

Reply:
+1 on clarifying that. Your attempt looks good to me.

Reply:
+1